apps/worker/src/app/workflow/usecases/send-message/digest/get-digest-events.usecase.ts

```diff
@@ -24,30 +22,12 @@ const LOG_CONTEXT = 'GetDigestEvents';
 export abstract class GetDigestEvents {
   constructor(protected jobRepository: JobRepository, private createExecutionDetails: CreateExecutionDetails) {}
 
-  protected getJobDigest(job: JobEntity): {
```

Comment:
this function is not get-digest-events related and we can move to utils/digest in order to reuse it.